manifest.toml

```diff
@@ -44,8 +44,8 @@ ram.runtime = "50M"
     [resources.sources]
 
         [resources.sources.main]
-        url = "https://github.com/penpot/penpot/archive/refs/tags/2.1.1.tar.gz"
-        sha256 = "0912464158ab8cdbef384686886de95b3318d81105522c3eeb9d703fea87fbf0"
+        url = "https://github.com/penpot/penpot/archive/refs/tags/2.1.2.tar.gz"
+        sha256 = "a61b1e57256c8070906f6cda1aee27b9b484f9e83bfdfacfecca512c0850dd02"
```

Comment:
Penpot has a tendency of overwriting tags, I want to wait till they officially announce the release before merging.